indra/newview/llagent.cpp

```diff
@@ -1453,48 +1453,70 @@ LLVector3 LLAgent::getReferenceUpVector()
 	return up_vector;
 }
 
-
+#pragma optimize("", off)
```

Comment:
?

Reply:
> ?

I felt that I would forget to remove it :-)